core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -91,6 +91,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
     logError(processErr.toString)
 
 
+    if(!new File(hc.getConf.clusterInfoFile.get).exists()){
```

Comment:
@vpatryshev regarding style - good catch, forgot about that
regarding checking whether `hc.getConf.clusterInfoFile.get` returns `null` -> not necessary, it's set with default when initiating context if not set